plugins/woocommerce/tests/legacy/unit-tests/order/class-wc-tests-order-functions.php

```diff
@@ -103,43 +104,28 @@ public function test_wc_orders_count() {
 
 		// Fake some datastores and order types for testing.
 		$test_counts = array(
-			'order'           => array(
-				array( OrderInternalStatus::ON_HOLD, 2 ),
-				array( OrderStatus::TRASH, 1 ),
+			'shop_order'      => array(
```

Comment:
Not sure how we feel about using `OrderInternalStatus` vs `OrderStatus` or if there are specific guidelines on when we should use each.